civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -227,10 +227,17 @@
     width: 100%;
   }
 }
-
-.AboutProjects-recent-activity {
-  padding: 20px; /*as with description-details consider maxwidth */
-
+/* events section overrides from Event page style to About Project page style */
+.AboutProjects-events {
+  h1 {
+    font-size: 1.5rem;
+  }
+  .EventCard-dateheader h4 {
+    font-size: 1.25rem;
+  }
+}
+.AboutProjects-recent-activity, .AboutProjects-events  {
```

Comment:
Just a quick change, this section was previously uniform 20px padding on all 4 sides, but the other sections on page are 25 top/bottom, 10 left/right, so now they should all do that including the new Events section.